unity-renderer/Assets/DCLFeatures/CameraReel/ScreenshotViewer/Scripts/ScreenshotViewerInfoSidePanelView.cs

```diff
@@ -80,5 +81,28 @@ public void SetPictureOwner(string userName, string avatarPictureUrl)
             photoOwnerNameLabel.text = userName;
             photoOwnerAvatarPicture.SetImage(avatarPictureUrl);
         }
+
+        private void ClearCurrentProfiles()
+        {
+            foreach ((_, PoolableObject poolObj) in profiles)
+                poolObj.Release();
+
+            profiles.Clear();
+        }
+
+        private Pool GetProfileEntryPool()
+        {
+            var poolId = $"PictureDetailProfile_{name}_{GetInstanceID()}";
```

Comment:
Same question regarding ID naming

Reply:
There is only one instance of `ScreenshotViewerInfoSidePanelView`. Anyway, hypothetically all instances could share the same pool. I will rename it.